pykern/pkcli/github.py

```diff
@@ -87,26 +87,22 @@ def repo_arg(self, repo):
             pkcli.command_error("repo argument not supplied")
         return self.repo(repo) if isinstance(repo, str) else repo
 
-    def _iter_subscriptions(self):
-        """Returns a list so that we don't get rate limited at startup."""
-
-        def _subscriptions():
-            if cfg.test_mode:
-                return [self._github.repository(r[0], r[1]) for r in _TEST_REPOS]
-            return self._github.subscriptions()
-
+    def list_org_repos(self, org, include_forks):
+        """Returns list of repos for org"""
         self.login()
+        o = self._github.organization(org)
```

Comment:
looks like `o` only used once. Maybe:
```python
for r in self._github.organization(org).repositories():
...
```
?

Reply:
Yes unused. I'll try to fix if I need to go back in. (That's the problem with merging before the review...)

Reply:
Fixed this in #421